### DISCUSSION TITLE
Query-Efficient Black-box Attack

### DISCUSSION THREAD

**Discussion by sroshan89** [Trusted-AI/adversarial-robustness-toolbox#1478] [General]
Hello,
I'm checking the available black-box evasion attacks in ART. In the Wiki, it's mentioned that the following attack is supported  [Query-Efficient Black-box Adversarial Examples](https://arxiv.org/abs/1712.07113). But I couldn't locate the implementation in the repository. 
I'd be thankful if you could provide a pointer to the code.

**Comment by beat-buesser** [comment]
Hi @sroshan89 I think this black-box method has been implemented as a seprate estimator in https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/main/art/estimators/classification/query_efficient_bb.py. It provides black-box gradient estimations in method `loss_gradients` that can be used with all white-box attacks.
